plugins/ua_config_default.c

```diff
@@ -823,57 +841,45 @@ UA_ClientConfig_setDefault(UA_ClientConfig *config) {
         config->eventLoop->registerEventSource(config->eventLoop, (UA_EventSource *)udpCM);
     }
 
-    if(config->sessionLocaleIdsSize > 0 && config->sessionLocaleIds) {
```

Comment:
It's not clear to me in what circumstances a config with sessionLocaleIds would be presented.  And it seemed heavy-handed to actively delete them in a "setDefault" function.

Reply:
True.
We don't need that.